tests/unit/config/test_percentile_distribution.py

```
@@ -0,0 +1,158 @@
+        with pytest.raises(ValueError):
+            ADAPTER.validate_python({"p50": 50000, "p99": float("inf")})
+
+    @pytest.mark.parametrize(
+        "p50,p99",
+        [
+            param(1, 1e40, id="overflow_ratio"),
+            param(100, 1e35, id="inf_implied_mean"),
+        ],
+    )  # fmt: skip
+    def test_extreme_ratio_rejected(self, p50: float, p99: float) -> None:
+        with pytest.raises(ValueError, match="too extreme|finite"):
```

Ruff (0.15.20)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)